core/src/worker/activities/local_activities.rs

```diff
@@ -564,6 +594,16 @@ impl LocalActivityManager {
         }
     }
 
+    pub(crate) fn get_nonfirst_attempt_count(&self, for_run_id: &str) -> usize {
+        let dlock = self.dat.lock();
+        dlock
+            .la_info
+            .iter()
+            .filter(|(id, info)| id.run_id == for_run_id && info.first_wft_has_ended)
```

Comment:
Are we concerned with local activities that were started for this run ID but their workflow was previously evicted?
Do we keep those around? If so, can we detect them?

Reply:
If that run was evicted then those LAs are invalidated and need to start over again anyway